### PR TITLE
Shorten k8s port name for embedded journal

### PR DESCRIPTION
**Pull request by jiacheliu3**
The port name for embedded journal was `embedded-journal`. K8s has a restriction for a port name to be shorter than 15 characters, so the Pod creation will fail. This changes the port name to be short enough to be accepted.

### CODE REVIEW

**Comment by jiacheliu3** [comment]
@madanadit That'll do. I've changed the name.

File: integration/kubernetes/helm/alluxio/templates/alluxio-master.yaml [hunk 1]
```diff
@@ -48,7 +48,7 @@ spec:
     name: web
   {{- if $isEmbedded }}
   - port: 19200
-    name: embedded-journal
+    name: ej
```
Comment (madanadit) [inline integration/kubernetes/helm/alluxio/templates/alluxio-master.yaml]: ^

File: integration/kubernetes/multiMaster-embeddedJournal/alluxio-master.yaml.template [hunk 2]
```diff
@@ -25,7 +25,7 @@ spec:
   - port: 19999
     name: web
   - port: 19200
-    name: embedded-journal
+    name: ej
```
Comment (madanadit) [inline integration/kubernetes/multiMaster-embeddedJournal/alluxio-master.yaml.template]: ^

File: integration/kubernetes/multiMaster-embeddedJournal/alluxio-master.yaml.template [hunk 3]
```diff
@@ -123,7 +123,7 @@ spec:
...
```
Comment (madanadit) [inline integration/kubernetes/multiMaster-embeddedJournal/alluxio-master.yaml.template]: ^

File: integration/kubernetes/multiMaster-embeddedJournal/alluxio-master.yaml.template [hunk 4]
```diff
@@ -221,7 +221,7 @@ spec:
...
```
Comment (madanadit) [inline integration/kubernetes/multiMaster-embeddedJournal/alluxio-master.yaml.template]: ^

File: integration/kubernetes/helm/alluxio/templates/alluxio-master.yaml (hunk 1 above)
Comment (jiacheliu3) [reply integration/kubernetes/helm/alluxio/templates/alluxio-master.yaml]: Sorry overlooked some of them. These have been changed now.

**Comment by gpang** [comment]
alluxio-bot, merge this please

**Pull request merged**